src/Game.cpp

```diff
@@ -12,8 +12,12 @@ std::string Game::getName() const
 
 int Game::score() const
 {
-    //needs implementation after made convertValueToPairOfIntsFrame()
-    return 0;
+    int partialScore = 0;
+    for (int i=0; i<10; i++)
+    {
+        partialScore = partialScore + frame_[i].getValue();
```

Comment:
Here you can use short form `partialScore += frame _[i].getValue();`

Reply:
Done